File: src/main/java/ee/kert/frontend/frontend/src/components/Item.js
function Item(props){
    return(
        <div>
            <div className="itemName">{ props.name}</div>
            <div className="itemPrice">{props.price}</div>
            <div className="itemCategory">{props.category}</div>
        </div>
    );
}
//muudatus
export default Item;

File: src/main/java/ee/kert/backend/model/Item.java
package ee.kert.backend.model;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;
import org.springframework.stereotype.Component;

import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.Id;
//muudatus
@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
@Entity
@Component

public class Item {
    @Id
    @GeneratedValue
    private Long id;
    private String name;
    private int price;
    private String category;
}
